docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+services:
+  scanpix-server:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    image: scanpix-server:1.0
+    ports:
+      - "5001:5001"
+    volumes:
+      - "./data:/scanpix/data"
```

Comment:
Can you bring up both dockers as services in this compose file?

Reply:
Should we dockerise the frontend though? Lets keep the electron app running locally.

Reply:
Yeah, makes sense. Just add a bash script that brings up the backend and runs the front end. Good to have a single command, I think.